CMakeLists.txt

```diff
@@ -0,0 +1,294 @@
+cmake_minimum_required(VERSION 3.14 FATAL_ERROR)
+
+set(CMAKE_CXX_STANDARD 17)
+set(CMAKE_CXX_STANDARD_REQUIRED ON)
+
+project(bladebit)
+
+set(CMAKE_CONFIGURATION_TYPES release debug)
```

Comment:
Maybe `Release` and `Debug` for consistency as they're the names used all across CMake projects?